qubes/mgmt.py

```diff
@@ -534,3 +570,95 @@ def vm_kill(self):
         assert not self.arg
         self.fire_event_for_permission()
         yield from self.dest.kill()
+
+    @api('mgmt.Events', no_payload=True)
```

Comment:
While at it - I'd like to abandon this part of API:
```
Normally server will not close the connection.
A method yielding events will not ever return a OK or EXCEPTION message.
```

Reason: a method could progress reports as events. This could be especially useful for backup-related calls, but also for VM clone etc.
Alternative: broadcasting progress reports as "normal" events. But in some cases it would be hard to correlate progress reports with actual operation.